common_factories.py

```diff
@@ -151,9 +151,15 @@ def addPostTests(f_quick_build):
             doStepIf=lambda step: savePackage(step) and hasEco(step),
         )
     )
+    if tmp_cleanup:
+        tmp_path = "/tmp/*"
```

Comment:
please use `mtr_tmp_path` to be more transparent where this comes from and it should only be `/tmp/var_auto_*`, we cannot remove the content of `/tmp`.